self_hosting_machinery/webgui/static/index.html

```diff
@@ -81,6 +82,8 @@
 
   <div class="container-xxl mt-5">
     <div class="main-tab-content">
+      <div class="main-tab-pane" id="stats">
+      </div>
```

Comment:
I thought that we add it dynamically. Why we need to add it here?

Reply:
it's not. Take a look how other tabs are defined
```
  <div class="container-xxl mt-5">
    <div class="main-tab-content">
      <div class="main-tab-pane" id="stats">
      </div>
      <div class="main-tab-pane" id="model-hosting">
      </div>
      <div class="main-tab-pane" id="upload">
      </div>
      <div class="main-tab-pane" id="finetune">
      </div>
      <div class="main-tab-pane" id="server-logs">
      </div>
      <div class="main-tab-pane" id="access-control">
      </div>
      <div class="main-tab-pane" id="settings">
      </div>
      <div class="main-tab-pane" id="license">
      </div>
    </div>
  </div>

```